app/models/shipit/deploy_spec/file_system.rb

```diff
@@ -32,7 +32,7 @@ def cacheable_config
           'merge' => {
             'require' => pull_request_required_statuses,
             'ignore' => pull_request_ignored_statuses,
-            'timeout' => pull_request_timeout.to_i,
```

Comment:
I'd like Shipit to remain MRI 2.2 compatible for a little more. But ultimately yes.